src/cr/cube/matrix/measure.py

```diff
@@ -982,6 +991,63 @@ def _subtotal_rows(self):
         )
 
 
+class _ScaledCounts(_BaseSecondOrderMeasure):
+    """Provider of scaled count based marginals"""
```

Comment:
Well, I have to say I am starting to come around to the idea of having marginals each having a distinct class, like measures do. These could share a common base class if they needed to share computations, like retrieving numeric values or something. That way you can have some diversity in the interface, like you could ask for the base-values separately from the subtotals if you needed to, which some other measures might and perhaps already do.

Also it would make calling them prettier, like:

```python
assemble_column_vector(measures.rows_margin.blocks)
```

or

```python
rows_scale_median = self._measures.rows_scale_median
return (
    None
    if rows_scale_median.is_not_computable_or_whatever
    else assemble_column_vector(rows_scale_median.blocks)
)
```

Not sure this whether this is the right moment, but as I'm seeing it shape up I'm starting to feel the benefit that would bring.

Maybe `.is_not_computable_or_whatever` could be `.has_blocks` or maybe `.is_defined` and that would be applicable to other margins as well without having to know the details of why it might undefined in this particular case.